assets/facets.js

```diff
@@ -7,12 +7,40 @@ class FacetFiltersForm extends HTMLElement {
       this.onSubmitHandler(event);
     }, 500);
 
-    this.querySelector('form').addEventListener('input', this.debouncedOnSubmit.bind(this));
+    const facetForm = this.querySelector('form');
+    facetForm.addEventListener('input', this.debouncedOnSubmit.bind(this));
+    facetForm.addEventListener('keydown', (event) => {
+      if (event.target.tagName === 'LI' && (event.code.topUpperCase() === 'SPACE' || event.code.toUpperCase()  === 'ENTER' )) {
+        event.preventDefault();
+        event.target.querySelector('input').click();
+      }
+    });
+
+    facetForm.addEventListener('click',  (event) => {
```

Comment:
I added this because before when you selected a filter the `Show more` cease to function. This was due to the DOM being rewritten and the events being removed.

Reply:
I also grouped them since this was an accessibility concern on windows: https://github.com/Shopify/dawn/pull/1443/commits/a9207a288723cc5ed5d06b6afed8515d7738601c

Reply:
Ah yes you're right, since the DOM nodes inside the form are being replaced, the event listeners (if applied to the button) would need to be re-attached. My suggestion would be to leverage custom HTML elements. The `connectedCallback()` lifecycle method allows event listeners to automatically re-attach when an instance of the custom element is added to the DOM.

I've done some testing and an implementation like this should work for this scenario: https://codepen.io/KaichenWang/pen/eYyvmqv

Reply:
Just saw this but I moved it out in its own component. 👀 :

https://github.com/Shopify/dawn/pull/1443/commits/3406756dbb923e2335cc15aa7a422c8ce519441e 👀 

I can include `connectedCallback`. Thoughts?

Reply:
I think your implementation with custom HTML element is good. In this case, attaching the event listeners in the `constructor()` also works.